tests/e2e/pageObjects/pub-sub-page.ts

```diff
@@ -49,16 +49,4 @@ export class PubSubPage {
         await this.publishMessage(channel, message);
         await t.expect(this.pubSubPageContainer.find('[data-testid^=row]').withText('message').exists).ok('Message is not displayed');
     }
-
-    /**
-     * Verify message is/not displayed
-     * @param message The text of message
-     * @param displayed True if should be displayed, false if shouldn't
-     */
-    async verifyMessageDisplaying(message: string, displayed: boolean): Promise<void> {
```

Comment:
Why was it removed?
I see you used it on various tests.
The only thing is that the location here is probably wrong.

Reply:
I moved it directly in tests, to not store it into pageObjects

Reply:
But now it's on each test. What will happen when you'll have modified it? Will you do that X times?
Add it somewhere on `tests/e2e/helpers`

Reply:
Yes, it makes sense, thanks, fixed